internal/dag/httpproxy_processor.go

```diff
@@ -144,6 +144,10 @@ func (p *HTTPProxyProcessor) computeHTTPProxy(proxy *contour_api_v1.HTTPProxy) {
 
 	var tlsEnabled bool
 	if tls := proxy.Spec.VirtualHost.TLS; tls != nil {
+		/*if tls.Passthrough && tls.EnableFallbackCertificate {
```

Comment:
@jpeach Where should I add a unit test for this? 

```
./internal/xdscache/v3/route_test.go
./internal/envoy/v3/route_test.go
./internal/featuretests/v3/route_test.go
```

Reply:
`./internal/dag/builder_test.go` is actually the spot to go, you'll see tests for all the processors in there. Probably we need to fix that up, but that's the spot to start.

Reply:
Do we have somewhere we test for error conditions we set while processing objects? I haven't been able to find much

Reply:
> Do we have somewhere we test for error conditions we set while processing objects? I haven't been able to find much

`internal/dag/status_test.go` has some of those, and then `internal/featuretests` do as well.

Reply:
Thanks. Added a test to `status_test`